applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp

```diff
@@ -267,17 +259,17 @@ namespace Kratos
     void LineLoadCondition2D::CalculateAndAddPressureForce(
         Vector& rRightHandSideVector,
         const Vector& N,
-        array_1d<double, 2>& Normal,
+        const array_1d<double, 3>& Normal,
         double Pressure,
         double IntegrationWeight 
         )
     {
-        const unsigned int NumberOfNodes = GetGeometry().size();
-        const unsigned int Dimension = 2;
+        const unsigned int number_of_nodes = GetGeometry().size();
+        const unsigned int dimension = 2;
```

Comment:
The name of the condition is line_load_condition_2d, we already decided that in the past